Api/src/guild-module-level/dto/update-guild-module-level.input.ts

```diff
@@ -1,8 +1,33 @@
 import { CreateGuildModuleLevelInput } from './create-guild-module-level.input';
-import { InputType, Field, Int, PartialType } from '@nestjs/graphql';
+import { Field, InputType, Int, PartialType } from '@nestjs/graphql';
 
 @InputType()
-export class UpdateGuildModuleLevelInput extends PartialType(CreateGuildModuleLevelInput) {
+export class UpdateGuildModuleLevelInput extends PartialType(
```

Comment:
When u extend a create DTO, all parameters are inherited. Thus, only ID should be needed to be added.

Reply:
Looks great!